src/components/wishlistCard/wishListCard.css

```diff
@@ -0,0 +1,17 @@
+.MuiContainer-root.removed-item{
+  animation: removed-item-animation .6s cubic-bezier(.55,-0.04,.91,.94) forwards;
+  clear:right;
+}
+
+@keyframes removed-item-animation {
+  from {
+      opacity: 1;
+      transform: scale(1);
+  }
+
+  to {
+      opacity: 0;
+      height: 0px;
+      transform: scale(0);
+  }
+}
```

Comment:
@RohankumarM also put this CSS into the index file of your code as we are not using external CSS files, put it inside makeStyles.

Reply:
Yes, I was busy updating the code. I have added inline styles and removed .css file. 

Was figuring out how to add keyframes and conditional classnames. 

I have reduced the delay as well. Seems good to me now. How about you? Needs to decrease the delay further?

Reply:
Yes !! @RohankumarM code looks good to me but I think you forget to delete that external CSS file as you add CSS to the "makeStyles" we don't need that external file you can remove that.